lib/v2/penguin-random-house/articles.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./utils');
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const link = 'https://www.penguinrandomhouse.com/articles';
```

Comment:
```suggestion
    const link = 'https://www.penguinrandomhouse.com/articles/';
```
See https://github.com/DIYgod/RSSHub/runs/7478958103?check_suite_focus=true#step:11:51